### DISCUSSION TITLE
Is modern (module/no-module) enabled by default?

### DISCUSSION THREAD

**Discussion by IskanderMustafin** [vercel/next.js#18567] [Help] [answered]
In [Next.js Conf – Keynote](https://youtu.be/1-NzQ9ObsfM?t=971) @Timer mentioned that differential loading for modern browsers is enabled by default. But according to source code of [next.config.js](https://github.com/vercel/next.js/blob/canary/packages/next/next-server/server/config.ts#L55) this configuration option is still experimental and it's default value is `false`. 

So I'm a bit confused and wondering should I explicitly define `experimental: { modern: true }` in my `next.config.js`?

**Comment by timneutkens** [comment, marked as answer]
Joe was referring to differential loading of polyfills. The modern mode experiment is still an experiment until it holds better results.

**Reply by neckaros** [reply]
How do you do differential loading of polyfills without module/nomodule pattern?

**Reply by rashidul0405** [reply]
> Joe was referring to differential loading of polyfills. The modern mode experiment is still an experiment until it holds better results.

@timneutkens I guess though bundle size doesn't improve that much but code executes faster on modern mode?

**Reply by Timer** [reply]
Our differential loading of polyfills does use the nomodule pattern.

**Reply by rashidul0405** [reply]
> Our differential loading of polyfills does use the nomodule pattern.

@Timer My question is a bit irrelevant. I've following the modern mode for a long time to be out of the experimental phase. I guess in modern mode, though bundle size doesn't improve that much but code executes faster? Am I right? Any idea/ETA of finalizing that part?
